src/components/ui/ProgressPanel.svelte

```diff
@@ -1,8 +1,8 @@
 <Panel title="{siteName || TRANSLATED.HEADING_PANEL}" subtitle="{siteName ? TRANSLATED.REPORT_FOR : ''}" bind:open="{panelIsOpen}">
 
-  <p class="your-report__description">Reported on {totalEvaluated} of {totalToEvaluate} WCAG {wgacVersion} {conformanceTarget} Success Criteria.</p>
+  <p class="your-report__description">{TRANSLATED.REPORTED_ON} {totalEvaluated} {TRANSLATED.REPORTED_ON_OF} {totalToEvaluate} WCAG {wcagVersion} {conformanceTarget} Success Criteria.</p>
```

Comment:
May want to concatenate this in script section to e.g. `TRANSLATED.REPORTED_DESCRIPTION` for easier reading.